### ISSUE TITLE
Fun new strategy idea!

### ISSUE THREAD

**Issue by jaypantone** [PermutaTriangle/Tilings#508]
Suppose we have a tiling A with 1 row, 3 cols, rightmost cell tracked
```
+---+---+---+
| 1 | 2 | 3 |  
+---+---+---+
```
We can have a strategy that deletes the tracked cell, creating child B, and has the reverse rule where the parent is B and the child is A.
```
+---+---+
| 1 | 2 | 
+---+---+
```
The GF is `B(x) = A(x,0)`. Counts are easy too because `b_n = a_{n,0}`.

We've never had a strategy like this before and I think it has a chance of linking together previously unlinked tilings. Equations will now have `0` and `1` substitutions which is weird and fun. Could be great for 1x5s. Might make reverse positive fusion work too.

**Comment by jaypantone** [comment]
Another possible idea.... turn the middle cell into a single point instead of zero points. This is like a partial point placement, but then of course you could place the point to get a normalish looking tiling.

**Comment by ulfarsson** [comment]
Christian told me that this is actually very similar to what happens when we are adding an Sk somewhere on top of the tiling. This is the case of adding S1 into this particular cell

**Comment by ulfarsson** [comment]
An extreme version of this idea would be: given tiling T, add an assumption (perhaps over multiple cells) to get the tiling T'. Now you have a rule T->T'. Now you can apply Jay' strategy to this new assumption (make the cells in it empty) to get a new tiling T'' and a rule T''-> T'. Note that this does not connect T and T'' directly, because both of the new rules are one-way.

**Comment by christianbean** [comment]
Implemented by PR #513
